static/src/stylesheets/module/_adslot.scss

```diff
@@ -528,3 +528,41 @@
     }
 
 }
+
+.mobilesticky-container {
```

Comment:
Should this classname be more BEM? Is this a `-container` or a `--container`? Same for the others here.

Reply:
This particular one is totally BEM but the others need a second pass. Cheers.